pybtp/btp.py

```diff
@@ -3353,6 +3353,10 @@ def gap_passkey_disp_ev_(gap, data, data_len):
 
     logging.debug("passkey = %r", passkey)
 
+    # unpacking passkey to int loses leading 0s,
+    # let's add them back if lost
+    passkey = str(passkey).zfill(6)
```

Comment:
Let's move debug log after this operation as the result will be actually used later.

Reply:
I agree, done